src/main/java/org/example/bookstore/repository/role/RoleRepository.java

```diff
@@ -0,0 +1,8 @@
+package org.example.bookstore.repository.role;
+
+import org.example.bookstore.model.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    Role getRoleById(Long id);
```

Comment:
* we can not really guarantee that we will always have a result
* I prefer to use `find` instead of get wich implies that we are **trying** to get it but d not guarantee the result, when **get** implies that we are certain that we will get what we are looking for 
* how will you know the id to fetch the role by? Hardcode is not the best scenario because we can have different id generation strategies and types, but role name will always be constant
```suggestion
    Optional<Role> findRoleByName(RoleName name);
```